CONTRIBUTING.md

```diff
@@ -91,6 +91,8 @@ for how to set up your development environment, or ask in #hackers-test on Disco
 Developing for Flutter
 ----------------------
 
+### Framework & Engine
```

Comment:
for consistency with the rest of the doc, let's use "and" here and in other new prose rather than "&".